etna/models/mixins.py

```diff
@@ -288,8 +288,7 @@ def _make_predictions_segment(
         model: Any, segment: str, ts: TSDataset, prediction_method: Callable, **kwargs
     ) -> pd.DataFrame:
         """Make predictions for one segment."""
-        segment_features = ts[:, segment, :]
-        segment_features = segment_features.droplevel("segment", axis=1)
+        segment_features = ts.to_pandas()[segment]
```

Comment:
I have doubts that it will slow down all our per-segment models because conversion `to_pandas` will happen for each segment and it isn't free (we make a copy where).

Reply:
May be we can make this copy only once in `_make_prediction` and remember it.